### ISSUE TITLE
Improving Moderator Selection

### ISSUE THREAD

**Issue by SamPatt** [OpenBazaar/OpenBazaar-Client#1001]
Currently moderator selection - both for vendor and buyer - takes place by selecting moderators from a list.

Unfortunately there is no other information presented about the moderator but the name and avatar. Clicking on the list selects them instead of taking you to their page. The current design creates an incentive for the user to select a moderator more or less at random, or to forego the moderation system altogether.

This is problematic for several reasons. 
1. The user doesn't know which language(s) the moderator speaks.
2. The user doesn't know the legal jurisdiction the moderator is in (important because some goods/services might be legal or illegal in certain countries).
3. The user doesn't know how long the moderator has been active, or how many cases they've settle, or any reputation aspects.
4. The user only knows the dispute fee %, but nothing else about them; their qualifications or their terms of service or their rules for dispute resolution.

Bottom line: There isn't enough information in the current moderator selection process for vendors or buyers to make an informed decision.

**Comment by morebrownies** [comment]
I can put together better designs for this experience, but I'll need to know from @cpacia what kind of data I'll have access to. Otherwise I'll be guessing or making assumptions that may not be feasible.
